apps/web/src/views/BuyCrypto/constants.ts

```diff
@@ -9,11 +9,28 @@ export const ETHEREUM_TOKENS = ['ETH', 'USDT', 'DAI']
 export const SUPPORTED_MONPAY_ETH_TOKENS = ['eth', 'usdc', 'dai', 'usdt']
 export const SUPPORTED_MOONPAY_BSC_TOKENS = ['bnb_bsc', 'busd_bsc']
 export const SUPPORTED_MERCURYO_FIAT_CURRENCIES = ['USD', 'EUR', 'GBP', 'HKD', 'CAD', 'AUD', 'BRL', 'JPY', 'KRW', 'VND']
+const MOONPAY_FEE_TYPES = ['Total Fees', 'Networking Fees', 'Provider Fees']
```

Comment:
i would prefer to leave this as an array, because we need to interate over this eventually. i could change to enum to make this look more clean, but we will be adding complexity when i need to interate over the fee types cos ill need to do something like Object.values(MOONPAY_FEE_TYPES).MAP(....) instead of just MOONPAY_FEE_TYPES.map(...)

Reply:
We could keep the array here and also define an enum like this
```typescript
enum FeeType {
  TotalFees = 'Total Fees',
  NetworkFees = 'Networking Fees',
  ...
}
const MOONPAY_FEE_TYPES = [FeeType.TotalFees, FeeType.NetworkFees]
const MERCURYO_FEE_TYPES = [FeeType.TotalFees]
```

The reason I suggest this way is because the 'Total Fees' value itself is a string literal while it's duplicated in both `MOONPAY_FEE_TYPES` and `MERCURYO_FEE_TYPES`. Which means if the value need to be updated, we need to do it in two places instead of one.

In practice, if we find same string literals being defined more than once, we better reconsider the way we use this variable

Reply:
We could keep the array here and also define an enum like this
```typescript
enum FeeType {
  TotalFees = 'Total Fees',
  NetworkFees = 'Networking Fees',
  ...
}
const MOONPAY_FEE_TYPES = [FeeType.TotalFees, FeeType.NetworkFees]
const MERCURYO_FEE_TYPES = [FeeType.TotalFees]
```

The reason I suggest this way is because the 'Total Fees' value itself is a string literal while it's duplicated in both `MOONPAY_FEE_TYPES` and `MERCURYO_FEE_TYPES`. Which means if the value need to be updated, we need to do it in two places instead of one.

In practice, if we find same string literals being defined more than once, we better reconsider the way we use this variable